rpc/session_foms.c

```diff
@@ -403,13 +403,9 @@ M0_INTERNAL int m0_rpc_fom_session_establish_tick(struct m0_fom *fom)
 		session_state_set(session, M0_RPC_SESSION_IDLE);
 		reply->rser_session_id = session->s_session_id;
 
-		M0_MEAS("conn-to-session conn_uuid: "U128X_F" conn_sm_id: %"
-			PRIu64" session_sm_id: %"PRIu64, U128_P(&conn->c_uuid),
+		M0_MEAS("conn-to-session conn_sm_id: %"PRIu64
```

Comment:
think about formatting the string into yaml as it'll be easier to parse such format inside the script.